tool/Cargo.toml

```diff
@@ -23,3 +23,7 @@ toml = "0.5.8"
 
 [dev-dependencies]
 insta = { version = "1.7.1", features = [ "backtrace" ] }
+
+[dependencies.tera]
```

Comment:
we should use a normal `tera = { version = "1", default-features = false }` in the `[dependencies]` section instead of this

Reply:
Fixed.